src/hooks/use-axios.tsx

```diff
@@ -55,6 +55,7 @@ const useAxios = <
         const responseData = transform ? transform(res.data) : res.data
         setResponse(responseData as TransformedResponse)
         setError(null)
+        // eslint-disable-next-line @typescript-eslint/no-floating-promises
         onResponse && onResponse(responseData as TransformedResponse)?.catch()
```

Comment:
I think we can add `void` before calling the `onResponse` function and delete the `catch` invocation

Reply:
As far as I remember if we add void then SonarCloud won`t be happy about that